sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/service/dynamicconfig/config/DynamicConfig.java

```diff
@@ -83,7 +83,7 @@ public class DynamicConfig implements BaseConfig {
     private boolean enableAuth = false;
 
     /**
-     * 请求超时时间，nacos每次getConfig需要设置一个超时时间
+     * Request timeout. Nacos needs to set one timeout per get config
```

Comment:
Nacos needs to set one timeout per getConfig

Reply:
done